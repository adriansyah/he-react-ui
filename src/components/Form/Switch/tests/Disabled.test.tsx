import createTestContext from 'react-cosmos-test/enzyme';
import fixture from '../fixtures/Disabled.fixture';

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

test('<Switch /> rendered correctly with disabled fixture', () => {
  const wrapper = getWrapper();

  expect(wrapper).toMatchSnapshot();
});
