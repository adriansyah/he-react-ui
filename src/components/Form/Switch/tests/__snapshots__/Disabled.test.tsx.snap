// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Switch /> rendered correctly with disabled fixture 1`] = `
<Switch
  checked={false}
  disabled={true}
  error=""
  id="demo"
  label="Label"
  name="demo"
  onChange={[Function]}
>
  <div
    checked={false}
    className="outer disabled"
    error=""
    id="demo"
    name="demo"
    onClick={[Function]}
  >
    <div
      className="switchContainer"
    >
      <input
        className="toggleSwitch"
        type="checkbox"
      />
      <span
        className="toggleSlider"
      />
    </div>
    <span
      className="text"
    >
      Label
    </span>
  </div>
</Switch>
`;
